github-stats/fetcher/comments_test.go

```diff
@@ -0,0 +1,257 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/google/go-github/github"
+)
+
+func TestFindLatestCommentUpdate(t *testing.T) {
```

Comment:
That should fall in the same use-case as "We don't have any comment". It will return an empty list.